test/ssltest_old.c

```diff
@@ -1891,7 +1891,7 @@ int doit_localhost(SSL *s_ssl, SSL *c_ssl, int family, long count,
          */
 
         /*
-         * We have non-blocking behaviour throughout this test program, but
+         * We have non-blocking behavior throughout this test program, but
```

Comment:
And it doesn't make sense. If you are taking stand for this one, then you have to be even more categorical about "initialise" [because it's a) also in this request, b) also in commentary, c) as already mentioned you kind of have Oxford English on your side, and most importantly d) actually really needs correction]. But you don't... Too colorless?